src/Libraries/DSOffice/DSOffice.csproj

```diff
@@ -70,6 +68,12 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\DynamoCore\DynamoCore.csproj">
+      <Project>{7858fa8c-475f-4b8e-b468-1f8200778cf8}</Project>
+      <Name>DynamoCore</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
     <Content Include="DSOffice.Migrations.xml">
```

Comment:
I see no reason why I should see any changes to `DSOffice` here.  Please remove.